### PR TITLE
Add cloud logging

### PR DESCRIPTION
**Pull request by brianmodel**
Created custom logger that supports cloud logging with stackdriver. The logger also writes logs to a logs.txt file.

### CODE REVIEW

File: cmd/start_server/main.go [hunk 1]
```diff
@@ -15,9 +15,28 @@
...
+	"log"
+
+	"github.com/googleinterns/cloudai-gcp-test-resource-reaper/pkg/logger"
 	"github.com/googleinterns/cloudai-gcp-test-resource-reaper/pkg/manager"
 )
 
 func main() {
-	manager.StartServer("localhost", "8000")
+	port := flag.String("port", "8000", "port to run gRPC server on")
+	logsProject := flag.String("logs-project", "", "GCP Project ID for where to store logs")
```
Comment (supercclank) [inline cmd/start_server/main.go:28]: Nit: Should this be "GCP Project ID **in which** to store logs"?

File: cmd/start_server/main.go [hunk 2]
```diff
@@ -15,9 +15,28 @@
...
+	logsName := flag.String("logs-name", "", "name of logs")
```
Comment (supercclank) [inline cmd/start_server/main.go:29]: Can we give this a default?

File: cmd/start_server/main.go [hunk 3]
```diff
@@ -15,9 +15,28 @@
...
+
+	flag.Parse()
+
+	if err := logger.CreateLogger(); err != nil {
+		log.Fatal(err)
+	}
+	defer logger.Close()
+	if len(*logsProject) > 0 && len(*logsName) > 0 {
+		logger.AddCloudLogger(context.Background(), *logsProject, *logsName)
```
Comment (supercclank) [inline cmd/start_server/main.go:38]: Is it worth adding some logging here about which project is being logged to?

Probably warning level - I think we may also want to log at warning if no project is provided.

Thoughts?

File: cmd/start_server/main.go (hunk 2 above)
Comment (brianmodel) [reply cmd/start_server/main.go:29]: I thought cloud logging should be optional, so if no value is set then there will be no cloud logging

File: cmd/start_server/main.go (hunk 1 above)
Comment (brianmodel) [reply cmd/start_server/main.go]: Change flag to project-id, and changed variable name to projectID

File: cmd/start_server/main.go (hunk 3 above)
Comment (brianmodel) [reply cmd/start_server/main.go]: Added log message for which logs and project are being written to

**Pull request merged**